jsx/App/Stories/Story/lib/txt_sync.js

```diff
@@ -11,67 +36,104 @@ export function setupTextSync() {
         } 
     }
 
+    /* Sync function for files with AV */
     window.sync = function sync(current_time) {
         for (var i=0; i<ts_tag_array.length; i++) {
             // Somewhat hacky solution: decrease current_time by 0.001 to avoid highlighting before player starts
             if ((current_time-0.001 >= parseFloat(ts_start_time_array[i])/1000.0) && (current_time <= parseFloat(ts_stop_time_array[i])/1000.0)) {
                 ts_tag_array[i].setAttribute("id", "current");
-                // $('#example, #td').animate({scrollTop:$("#current").offset().top}, 500);
                 scrollIntoViewIfNeeded($("#current")[0]);
-                ts_tag_array[i].style.backgroundColor = "rgb(209, 200, 225)";
+                changeSentenceColorToHighlight(i); 
             }
             else {
-                ts_tag_array[i].style.backgroundColor = "transparent";
+                changeSentenceColorToNormal(i);
                 try { ts_tag_array[i].removeAttribute("id"); }
                 catch (err) { }
             }
         }
     }
 
-    try {
-        const media = document.querySelectorAll("[data-live='true']")[0];
-        media.setAttribute("ontimeupdate", "sync(this.currentTime)");
-        media.setAttribute("onclick", "sync(this.currentTime)");
-    } catch (err) {
-        console.log(err);
+    /* Two functions that change the color of a sentence to either highlight or normal. */
+    function changeSentenceColorToHighlight(timestampIndex) {
```

Comment:
This is kind of a nitpick, but can we rename this function? I keep thinking it means "change which of the sentence colors we're supposed to highlight". A name like changeSentenceColorToHighlight**ed** or even just highlightSentence  (and rename the other function to unHighlightSentence or something) would do the trick.

Reply:
Updated